docs/source/getting_started/index.rst

```diff
@@ -0,0 +1,33 @@
+Getting started
+===============
+
+.. include:: install.rst
+
+Next steps
+^^^^^^^^^^
+
+DoWhy is based on a simple unifying language for causal inference, unifying two powerful frameworks,
+graphical causal models and potential outcomes. It uses graph-based criteria and do-calculus for modeling assumptions
+and identifying a non-parametric causal effect.
+
+For estimation, it switches to methods based primarily on potential outcomes and the best place to learn more about
+this is the user guide's :doc:`../user_guide/effect_inference/index` chapter. Alternatively, learn about its basic
+features in :doc:`../examples_notebooks/dowhy_simple_example`.
```

Comment:
these links to notebooks (also for GCM below) are not rendering correctly somehow
https://petergtz.github.io/dowhy/master/getting_started/index.html

Reply:
Good catch. There's a typo (`examples_notebooks`). Fixed in:
- https://github.com/py-why/dowhy/pull/578